README.md

```diff
@@ -32,28 +32,30 @@ Winit is designed to be a low-level brick in a hierarchy of libraries. Consequen
 show something on the window you need to use the platform-specific getters provided by winit, or
 another library.
 
-```rust
+```rust, no_run
 use winit::{
     event::{Event, WindowEvent},
-    event_loop::{ControlFlow, EventLoop},
+    event_loop::EventLoop,
     window::WindowBuilder,
 };
 
-fn main() {
-    let event_loop = EventLoop::new();
-    let window = WindowBuilder::new().build(&event_loop).unwrap();
+fn main() -> Result<(), Box<dyn std::error::Error>> {
+    let event_loop = EventLoop::new().unwrap();
+    let window = WindowBuilder::new().build(&event_loop)?;
 
```

Comment:
Yeah, we should just remove this example in the README, there's no value and we have on in the `lib.rs`, so it's visible on `docs.rs` which is probably more important.

Reply:
There has not really been consensus whether that entirely the best idea, not that I know of at least, see e.g. [this question I asked on the Rust API guidelines repo almost two years ago](https://github.com/rust-lang/api-guidelines/discussions/254).

That said, I agree, it'd also be my preference to consolidate the documentation to be on docs.rs.